owl-bot-post-processor/main.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Run this script from the root directory of the repository, like this:
+#    $ ./owlbot-post-processor/main.sh
+
+# This script does two things:
+# 1. Copies files from the /owl-bot-staging directory.
+# 2. Invokes ../generate-projects.sh to generate .csproj files.
+
+set -ex
+
+TMP=$(mktemp -d)
+
+# Copies one api from the /owl-bot-staging directory into the the /apis dir.
+copy_one_api() {
+  PACKAGE=$1
+  STAGING_DIR=owl-bot-staging/$1
+  PACKAGE_DIR=apis/$1
+
+  # All source files need the extension .g.cs.  Some will only be generated with the
```

Comment:
Forgot to note that this should be removable really imminently. I'll keep an eye on it though - I suspect I'm competent to remove it when we can, and it won't do any harm anyway.